tests/test_error_cases.py

```diff
@@ -19,14 +21,12 @@ def compatible_catalogs(self):
             return "CatsimCatalog", "CosmosCatalog"
 
     with pytest.raises(AttributeError) as excinfo:
-        catalog_name = "data/sample_input_catalog.fits"
-
         stamp_size = 24.0
         batch_size = 8
         cpus = 1
         add_noise = True
 
-        catalog = btk.catalog.CatsimCatalog.from_file(catalog_name)
+        catalog = btk.catalog.CatsimCatalog.from_file(CATALOG_PATH)
```

Comment:
It seems like the unit tests are failing in this function. Could you take a look @aboucaud ? You might also need to rebase because I just pushed a change to fix a separate issue in the unit-tests. (I would help out with this myself but since this is in a fork I can't make the changes)

Reply:
You can because I gave you the rights as maintainer of the repo.